### PR TITLE
Can now long press on markdown links

### PR DESCRIPTION
**Pull request by csduarte**
#### Summary
This PR allows URL only posts to be long pressed.

#### Ticket Link
https://mattermost.atlassian.net/browse/RN-201

#### Device Information
This PR was tested on: [Device name(s), OS version(s)] 
This PR was tested on:
iPhone 6 iOS 10
iPhone 7 iOS 10
Galaxy s7 - Android 6.0 API 23

### CODE REVIEW

File: app/components/markdown/markdown_link.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ import CustomPropTypes from 'app/constants/custom_prop_types';
 export default class MarkdownLink extends PureComponent {
     static propTypes = {
         children: CustomPropTypes.Children.isRequired,
-        href: PropTypes.string.isRequired
+        href: PropTypes.string.isRequired,
+        onLongPress: PropTypes.func.isRequired
```
Comment (enahum) [inline app/components/markdown/markdown_link.js]: can we add this as a defaultProp and remove the `Required`?

I know this is being used in Channel Info to display the channel header

**Comment by jarredwitt** [comment]
@enahum done.

File: app/components/markdown/index.js [hunk 2]
```diff
@@ -30,7 +30,8 @@ export default class Markdown extends React.PureComponent {
         textStyles: PropTypes.object,
         blockStyles: PropTypes.object,
         emojiSizes: PropTypes.object,
-        value: PropTypes.string.isRequired
+        value: PropTypes.string.isRequired,
+        onLongPress: PropTypes.func.isRequired
```
Comment (enahum) [inline app/components/markdown/index.js]: @jarredwitt you missed it here

**Comment by jarredwitt** [comment]
@enahum done.

**Comment by jarredwitt** [comment]
@hmhealey I was just able to long press anywhere on the body of a text only post. Is that different than the behavior that you're seeing?

**Comment by hmhealey** [comment]
Never mind. I must've been confusing it with something I was seeing when my post was still pending.

**Pull request merged**